extensions/wikia/ARecoveryEngine/js/helper.js

```diff
@@ -81,8 +83,23 @@ define('ext.wikia.aRecoveryEngine.recovery.helper', [
 		}
 	}
 
+	function getRecoveredSlot(slotName) {
```

Comment:
Please move it to separate module because it's not used in Mercury and it will be easier to remove "recovery" dependencies from Mercury in future.